mrblib/mitamae/resource_executor/link.rb

```diff
@@ -3,7 +3,7 @@ module ResourceExecutor
     class Link < Base
       def apply
         unless run_specinfra(:check_file_is_linked_to, desired.link, desired.to)
-          run_specinfra(:link_file_to, desired.link, desired.to, force: desired.force)
+          run_specinfra(:link_file_to, desired.link, desired.to, force: desired.force, no_dereference: desired.force)
```

Comment:
Thanks for the comment. Because I can achieve my goal, the change that make new option is no problem for me. But the behavior of mitamae become different from itamae. Is it no problem? ref. https://github.com/itamae-kitchen/itamae/pull/229